tests/link/numba/test_slinalg.py

```diff
@@ -109,23 +108,25 @@ def test_solve_triangular_raises_on_nan_inf(value):
 
 
 @pytest.mark.parametrize("lower", [True, False], ids=["lower=True", "lower=False"])
-def test_numba_Cholesky(lower):
-    x = set_test_value(
-        pt.tensor(dtype=config.floatX, shape=(3, 3)),
-        (lambda x: x.T.dot(x))(rng.random(size=(3, 3)).astype(config.floatX)),
-    )
+@pytest.mark.parametrize("trans", [True, False], ids=["trans=True", "trans=False"])
+def test_numba_Cholesky(lower, trans):
+    cov = pt.matrix("cov")
 
-    g = pt.linalg.cholesky(x, lower=lower)
-    g_fg = FunctionGraph(outputs=[g])
+    if trans:
+        cov_ = cov.T
+    else:
+        cov_ = cov
+    chol = pt.linalg.cholesky(cov_, lower=lower)
 
-    compare_numba_and_py(
-        g_fg,
-        [
-            i.tag.test_value
-            for i in g_fg.inputs
-            if not isinstance(i, SharedVariable | Constant)
-        ],
-    )
+    fg = FunctionGraph(outputs=[chol])
+
+    x = np.array([0.1, 0.2, 0.3])
+    val = np.eye(3) + x[None, :] + x[:, None]
+
+    compare_numba_and_py(fg, [val])
+
+    func = pytensor.function([cov], chol, mode="NUMBA")
+    np.testing.assert_allclose(func(val), scipy.linalg.cholesky(val, lower=lower))
```

Comment:
Isn't `compare_numba_and_py` already checking numba and python implementations match?

Reply:
It should, but didn't find the problem because `compare_numba_and_py` calls the functions without fast_run rewrites. That excluded Blockwise, and because that doesn't have a numba implementation it ended up calling the python code for the numba and the python version.

I included the rewrite that remove useless Blockwise ops into the set of rewrites we do in testing.
Maybe we should actually just use the default numba mode?

Reply:
Ahh... okay so hopefully just a temporary blind spot. Let's keep it like you did now